BEIMA.Client/Dockerfile

```diff
@@ -0,0 +1,48 @@
+# Support until 2025
+from ubuntu:20.04
+# Set non interactive to avoid having to make keyboard/timezone selection
+ENV DEBIAN_FRONTEND noninteractive
+# Allow connections on port 3000
+EXPOSE 3000
+# Create directory for front end
+RUN mkdir /beima
+# Set work directory
+WORKDIR /beima
+# Update the apt package manager
+RUN apt update
+# Set time zone env variable and install tzdata package using it
+RUN TZ=Etc/UTC apt-get -y install tzdata
+# Move all required folders into the container
+COPY ./BEIMA.Client/ /beima/BEIMA.Client/
```

Comment:
We will need to make sure Dr. Henderson gets the local.settings.json and the .env added to our project before running this. It's been a while since I played with Docker, I'm glad we can copy over what we need and won't need to worry about getting those files added a different way

Reply:
I was thinking about that too. There's likely going to have to be a production version of at least the .env file to point at the DNS name of the api on the local docker network instead of localhost.

Reply:
production version/flags will be needed for local.settings.json, looks like the env file will be fine